lib/helper.js

```diff
@@ -146,30 +126,67 @@ const countPlayers = (lineup, type, state, rc) => {
 };
 
 /**
- * findNextPlayer
+ * findNextPlayer returns next players
  *
- * @param
- * @returns
+ * @param lineup
+ * @param palyerPos as index in lineup
```

Comment:
I am unsure of the function of @param palyerPos, but it looks like it is not being used